### PR TITLE
Update style.json

### PR DESCRIPTION
**Pull request by zstadler**
- Remove unnecessary references to `osm-liberty`
- Spelling corrections

### CODE REVIEW

File: style.json [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
     "openmaptiles:version": "3.x"
   },
   "sources": {
-    "fludpoints": {
+    "floodpoints": {
       "type": "geojson",
       "data": "https://raw.githack.com/HarelM/IHMMapboxMapExample/main/source.geojson"
     }
   },
-  "sprite": "https://maputnik.github.io/osm-liberty/sprites/osm-liberty",
```
Comment (HarelM) [inline style.json:15]: Are you sure a style will work with an empty sprite?

File: style.json [hunk 2]
```diff
@@ -1,25 +1,24 @@
...
+  "sprite": "",
   "glyphs": "https://api.maptiler.com/fonts/{fontstack}/{range}.pbf?key={key}",
```
Comment (HarelM) [inline style.json:15]: Probably better to change this as well, while you are at it...

**Review by HarelM — CHANGES_REQUESTED** [review]

File: style.json (hunk 1 above)
Comment (zstadler) [reply style.json:15]: [`sprite` is optional](https://docs.mapbox.com/mapbox-gl-js/style-spec/root/#sprite) and can safely be removed.
I think IHM overwrites it anyway, isn't it?

File: style.json (hunk 2 above)
Comment (zstadler) [reply style.json:15]: Change what to what?

File: style.json (hunk 1 above)
Comment (HarelM) [reply style.json:15]: Right... Forget that...

File: style.json (hunk 2 above)
Comment (HarelM) [reply style.json:15]: Orangemug?
Forget this is also ignored.
Maybe we can use our fonts...?

Comment (zstadler) [reply style.json:15]: Feel free to modify this PR as you like.

**Review by HarelM — APPROVED** [review]

**Pull request merged**